src/gp_doctest.cpp

```diff
@@ -766,3 +766,79 @@ TEST_CASE("GPInstance: Only add child node tests") {
   CHECK_EQ(dag.GetEdgeRange(dag.GetDAGNode(11)->GetBitset(), false).first, 3);
   CHECK_EQ(dag.GetEdgeRange(dag.GetDAGNode(11)->GetBitset(), false).second, 5);
 }
+
+// See diagram at: https://github.com/phylovi/bito/pull/366
```

Comment:
Please refer to any diagram directly, e.g. https://github.com/phylovi/bito/pull/366#issuecomment-922781415 if that's the one.